### ISSUE TITLE
Remove Q downloads and criteria? 

### ISSUE THREAD

**Issue by lindsayplatt** [lindsayplatt/episodic-river-salinization-model#8]
We were using Q for two purposes, neither of which may be necessary (depending on #5). Although, we would use it for #7.

1. Determining which days in the time series qualified as a "baseflow day" (no longer relevant)
2. Calculating the median streamflow for each site (could be obsolete if we get streamflow for each COMID from a different source via #5)

**Comment by lindsayplatt** [comment]
Won't fix for now because we may end up needing Q for the C-Q relationships (#7) or something else.
